File: fixtures/python_smoke/src/python_smoke/bench.py
from __future__ import annotations

import os

from .core import benchmark_metrics


def main() -> int:
    profile = os.environ.get("BENCH_PROFILE", "fast")
    metrics = benchmark_metrics(profile)
    print(f"STATUS: {metrics['status']}")
    print(f"TIME_MS: {metrics['time_ms']}")
    print(f"OPS_PER_SEC: {metrics['ops_per_sec']}")
    return 0


if __name__ == "__main__":
    raise SystemExit(main())

File: fixtures/node_smoke/scripts/bench.js
const { benchmarkMetrics } = require("../src");

const profile = process.env.BENCH_PROFILE || "fast";
const metrics = benchmarkMetrics(profile);

console.log(`STATUS: ${metrics.status}`);
console.log(`LATENCY_MS: ${metrics.latency_ms}`);
console.log(`OPS_PER_SEC: ${metrics.ops_per_sec}`);
